app/imports/api/actionItems/server/publications.js

```diff
@@ -5,21 +5,36 @@ import { isAdmin } from '/imports/api/users';
 import { Sprints } from '/imports/api/sprints';
 import { ActionItems } from './../ActionItems.js';
 
-Meteor.publish('actionItems', function publishActionItems(sprintId) {
-    check(sprintId, String);
+
+Meteor.publish('actionItems', function publishActionItems(sprintOrProjectId, type = 'sprintId') {
```

Comment:
I think that you can change also `type` to `isSprint = true`, because we have only two cases.